mne/surface.py

```diff
@@ -333,8 +333,8 @@ def complete_surface_info(surf, do_neighbor_vert=False, copy=True,
     if len(idx) > 0:
         logger.info('    Vertices [%s] have fewer than three neighboring '
                     'tris, omitted' % ','.join([str(ii) for ii in idx]))
-    for k in idx:
-        surf['neighbor_tri'] = np.array([], int)
+        for k in idx:
+            surf['neighbor_tri'][k] = np.array([], int)
```

Comment:
the relevant change is adding `[k]`, that was a bug if I'm not mistaken, as for the indentation it's doubtful whether it will save more time cumulatively than it took me to indent and commit :)